examples/ticks/index.js

```diff
@@ -1,25 +1,34 @@
-import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api";
+import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api/dist/DerivAPIBasic";
+
 const app_id = 1089; // Replace with your app_id or leave as 1089 for testing.
 const connection = new WebSocket(
   `wss://ws.binaryws.com/websockets/v3?app_id=${app_id}`
 );
 const api = new DerivAPIBasic({ connection });
+const tick_stream = api.subscribe({ ticks: "R_100" });
 
-const tickResponse = (res) => {
+const tickResponse = async (res) => {
   const data = JSON.parse(res.data);
-  console.log(data.tick);
-
-  // After 15 seconds, will stop tick response.
-  setTimeout(async () => {
+  if (data.error !== undefined) {
+    console.log("Error : %o", data.error.message);
     connection.removeEventListener("message", tickResponse, false);
     await api.disconnect();
-  }, 15000);
+  }
+  console.log(data.tick);
```

Comment:
Do we need this log? Seems like this might have been left unintentionally😄

Reply:
Good eye! This console.log is intended, although, I forgot to wrap it in an if statement so that it only gets logged when tick data gets requested. This is fixed now.